src/userSync.js

```diff
@@ -74,6 +75,8 @@ export function newUserSync(userSyncDependencies) {
       utils.logMessage(`Invoking image pixel user sync for bidder: ${bidderName}`);
       // Create image object and add the src url
       utils.triggerPixel(trackingPixelUrl);
+      // track that bidder has been synced
+      setBidderSynced(bidderName);
```

Comment:
@harpere for now, let's drop these as we are only storing keys for S2S bidders and not client side. Later on we can integrate them, possibly by appending `s2s${biddercode}` to differentiate.